v2/bindings/csharp/SnapPluginLibCs/CollectorExample/CollectorExample.cs

```diff
@@ -4,41 +4,43 @@
 
 namespace CollectorExample
 {
-    public class CollectorExample : CollectorPluginBase
+    public class CollectorExample : ICollectorPlugin
```

Comment:
Please provide streaming collector example

Reply:
Streaming collector and collector from code perspective has the same behavior. The only difference is when the metrics are published but this done under the hood.

Reply:
they do not, because in streaming you are expect to return from main function when the stream ends, so collector is closed. This is different in standard collect.